sdks/java/extensions/sql/zetasql/src/test/java/org/apache/beam/sdk/extensions/sql/zetasql/ZetaSqlDialectSpecTest.java

```diff
@@ -4067,4 +4064,113 @@ public void testSimpleTableName() {
             Row.withSchema(singleField).addValues(15L).build());
     pipeline.run().waitUntilFinish(Duration.standardMinutes(PIPELINE_EXECUTION_WAITTIME_MINUTES));
   }
+
+  /** Tests for logical_and operation with different inputs. */
+  @RunWith(Parameterized.class)
+  public static class LogicalAndTests {
+
+    @Parameterized.Parameters
+    public static Collection<Object[]> inputData() {
+      Object[][] data =
+          new Object[][] {{false, false, false}, {true, true, true}, {true, false, false}};
+      return Arrays.asList(data);
+    }
+
+    @Parameterized.Parameter(0)
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.